packages/manager/src/features/Linodes/LinodesDetail/LinodeSettings/LinodeSettingsMaintenancePolicyPanel.tsx

```diff
@@ -72,6 +78,12 @@ export const LinodeSettingsMaintenancePolicyPanel = (props: Props) => {
             {MAINTENANCE_POLICY_DESCRIPTION}{' '}
             <Link to={MAINTENANCE_POLICY_LEARN_MORE_URL}>Learn more</Link>.
           </Typography>
+          {showUpcomingMaintenanceNotice && (
+            <Notice variant="warning">
+              This Linode has an upcoming scheduled maintenance.{' '}
```

Comment:
```suggestion
              This Linode has upcoming scheduled maintenance.{' '}
```

Nit: Can remove the "an"?